services/QuillLMS/app/controllers/teachers/classroom_manager_controller.rb

```diff
@@ -269,4 +300,25 @@ def teacher_or_public_activity_packs
     end
   end
 
+  def set_alternative_schools
+    @alternative_schools = School.where(name: ['home school', 'us higher ed', 'international', 'not listed', 'other'])
+    @alternative_schools_name_map = {
+      'home school': 'Home school',
+      'us higher ed': 'U.S. higher education institution',
+      'other': 'Other',
+      'international': 'International institution',
+      'not listed': 'U.S. K-12 school'
+    }.stringify_keys
```

Comment:
These keys are already strings, so this isn't needed.

Reply:
fun fact: it turns out that if you use the hash rocket syntax, the key strings do get preserved!